test/conversations.spec.js

```diff
@@ -0,0 +1,193 @@
+const describe = require('mocha').describe;
```

Comment:
This line isn't necessary. `describe` will be available at runtime when executing `npm test`